data/hai/unfettered side missions.txt

```diff
@@ -65,3 +66,403 @@ mission "Unfettered returning home"
+			`	Her eyes light up as she focuses more on your ship. "I even see technology from those savage Korath that we fought valiantly to obtain!" `
+				to display
+					has "outfit (flagship installed): Korath Detainer"
+			`	All of a sudden, she raises her voice for everyone to hear. "I, Mera, challenge you! The winner will earn the right to own this honorable relic and its mighty weapons!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+				`	"I bought this ship. It's mine and I shall do what I please with it."`
+					goto headingout
+			`	"No, because in the eyes of my brethren that would mean I, too, did not deserve this ship. They would then challenge me for it."`
+			label win
+			`	Mera turns around, talking to other Unfettered for a bit before addressing you again. "You will win the right to access our worlds. But, in order to keep our respect, you must keep this Anomalocaris in good condition until you prove yourself another way. It is a priceless artifact. Should you refuse to prove your worth, you will no longer be allowed to land here.`
+			`	She then glares at you. "Beware, for although there are no official rules, you must use only the Anomalocaris. So, will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you. This is my ship now."`
+					goto headingout
+			`	Mera grins. "Good! I won't be fighting you personally, as it is too risky to do so. I am but an engineer, after all. I would also not want to deny my brother and his crew the opportunity to prove themselves against a mighty human warrior!"`
+			`	You're not sure if Mera is being serious or not, but her friends seem to be in for it. One that appears to be their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out," says a third.`
+			label scare
+			`	The leader of the group gets closer and addresses you in a more serious tone. "You should be aware that this is not so much a friendly competition as it is a fight for you to truly earn the right to fly this ship. You should know that this fight has a few... implicit rules, even if there are no official ones. We would assume everyone knows them, but since you are an outsider I will explain them to you," He says as he moves away from the group with you.`
+			`	"Naturally, this is a duel nobody other than us may interfere, but also know that being disabled counts as a loss. Don't even think about tricking us by switching to something other than this Anomalocaris, or we will not bother showing up! You must prove you deserve it, after all. To the victor go the spoils, so looting is allowed, but do not attempt to steal our ships. Do so, and all the Unfettered will be against you!"`
```

Comment:
```suggestion
			`	"Naturally, because this is a duel, nobody other than us may interfere, but you should also know that being disabled counts as a loss. Don't even think about tricking us by switching to something other than this Anomalocaris, or we will not bother showing up! You must prove you deserve it, after all. To the victor go the spoils, so looting is allowed, but do not attempt to steal our ships. Do so, and all the Unfettered will be against you!"`
```